strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-nuget-layout-provider/src/main/java/org/carlspring/strongbox/nuget/Nuspec.java

```diff
@@ -0,0 +1,603 @@
+/*
+ * Copyright 2019 Carlspring Consulting & Development Ltd.
+ * Copyright 2014 Dmitry Sviridov
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.carlspring.strongbox.nuget;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+import javax.xml.bind.*;
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlElementWrapper;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.adapters.XmlJavaTypeAdapter;
+import javax.xml.transform.sax.SAXSource;
+
+import org.carlspring.strongbox.nuget.metadata.ContentFile;
+import org.carlspring.strongbox.nuget.metadata.Dependencies;
+import org.carlspring.strongbox.nuget.metadata.DependenciesGroup;
+import org.carlspring.strongbox.nuget.metadata.Dependency;
+import org.carlspring.strongbox.nuget.metadata.File;
+import org.carlspring.strongbox.nuget.metadata.FrameworkAssembly;
+import org.carlspring.strongbox.nuget.metadata.PackageType;
+import org.carlspring.strongbox.nuget.metadata.Repository;
+import org.carlspring.strongbox.nuget.metadata.StringListTypeAdapter;
+import org.carlspring.strongbox.nuget.metadata.VersionTypeAdapter;
+import org.semver.Version;
+import org.xml.sax.InputSource;
+import org.xml.sax.SAXException;
+import org.xml.sax.XMLReader;
+import org.xml.sax.helpers.XMLReaderFactory;
+
+/**
+ * Class containing information about the NuGet package
+ * 
+ * @author sviridov
+ */
+@XmlRootElement(name = "package", namespace = Nuspec.NUSPEC_XML_NAMESPACE_2011)
+@XmlAccessorType(XmlAccessType.NONE)
+public class Nuspec implements Serializable
+{
+
+    /**
+     * Namespace for NuGet 2012 package specification
+     */
+    public static final String NUSPEC_XML_NAMESPACE_2012 = "http://schemas.microsoft.com/packaging/2011/10/nuspec.xsd";
```

Comment:
Cosmetic: Would you mind adding empty lines between fields? Thanks! :)

Reply:
This can also be added to the code style profiles for eclipse and idea.

The last version which @sbespalov provided should change the `blank_lines_before_field` setting value 0 -> 1 so the file should have this in the end:
`<setting id="org.eclipse.jdt.core.formatter.blank_lines_before_field" value="1"/>`

Reply:
Please, feel free to fix this in the code style and to raise a PR for the `strongbox-docs` with the updated Eclipse code style! Thanks! :)